sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/aggregation/Aggregation.java

```diff
@@ -55,7 +55,7 @@ MetricData toMetricData(
       Resource resource,
       InstrumentationLibraryInfo instrumentationLibraryInfo,
       InstrumentDescriptor descriptor,
-      Map<Labels, Accumulation> accumulationMap,
+      Map<Labels, ? extends Accumulation> accumulationMap,
```

Comment:
This will be transformed in T as well soon, but trying to limit the size of the PR.